test/integration/node-specific/client_encryption.test.ts

```diff
@@ -463,16 +463,10 @@ describe('ClientEncryption integration tests', function () {
     let client: MongoClient;
     let clientEncryption: ClientEncryption;
     let collection: Collection<DataKey>;
-    // Data Key Stuff
-    const AWS_ACCESS_KEY_ID = process.env.AWS_ACCESS_KEY_ID;
-    const AWS_SECRET_ACCESS_KEY = process.env.AWS_SECRET_ACCESS_KEY;
-    const AWS_REGION = process.env.AWS_REGION;
-    const AWS_CMK_ID = process.env.AWS_CMK_ID;
 
     const kmsProviders = {
-      aws: { accessKeyId: AWS_ACCESS_KEY_ID, secretAccessKey: AWS_SECRET_ACCESS_KEY }
+      local: { key: Buffer.alloc(96) }
```

Comment:
These tests still rely on expansions because these were node-specific unit tests.

I chose to rewrite these tests to be environment-independent because they're testing keyAltNames functionality, not AWS KMS integration.